src/index.js

```diff
@@ -1,51 +1,45 @@
 import './styles/style.css';
 
 const todoList = [
-    {
-    "index": "1",
-    "description": "Booking my flight tomorrow",
-    "completed": false
-},
-{
-    "index": "2",
-    "description": "Visiting my children school friday",
-    "completed": false
-},
-{
-    "index": "3",
-    "description": "Going for groceries shopping",
-    "completed": false
-},
-{
-    "index": "3",
-    "description": "Visiting the gym",
-    "completed": false
-}
+  {
+    index: '1',
+    description: 'Booking my flight tomorrow',
+    completed: false,
+  },
+  {
+    index: '2',
+    description: 'Visiting my children school friday',
+    completed: false,
+  },
+  {
+    index: '3',
+    description: 'Going for groceries shopping',
+    completed: false,
+  },
+  {
+    index: '3',
+    description: 'Visiting the gym',
+    completed: false,
+  },
```

Comment:
- [ ] As specified in the project, kindly make sure that the value for the `index` is of `number` instead of a string. See below for more details:
<img width="958" alt="Screenshot 2023-06-14 at 22 33 15" src="https://github.com/ctokoli/Todo-list-with-webpack/assets/107192076/5e4d7d98-3788-4427-b822-c24bfebd5372">